var/spack/repos/builtin/packages/singularityce/package.py

```diff
@@ -120,6 +117,8 @@ def perm_script_path(self):
 
     def _build_script(self, filename, variable_data):
         with open(filename, "w") as f:
```

Comment:
You might want to add a BlueBrain comment here and at line 204, to make sure this doesn't get lost when upgrading to spack@0.20

Reply:
My idea is that if you think my changes are good in the builtin `singularityce` package to create a PR in upstream spack with those changes as well

Reply:
OK, that makes the BlueBrain comment not needed :-)